sdk/src/main/java/io/opentelemetry/sdk/trace/SpanBuilderSdk.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.trace;
+
+import io.opentelemetry.internal.Utils;
+import io.opentelemetry.trace.Link;
+import io.opentelemetry.trace.Sampler;
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.Span.Builder;
+import io.opentelemetry.trace.Span.Kind;
+import io.opentelemetry.trace.SpanContext;
+import java.util.ArrayList;
+import java.util.List;
+
+/** {@link SpanBuilderSdk} is SDK implementation of {@link Span.Builder}. */
+public class SpanBuilderSdk implements Span.Builder {
+
+  private Span parent;
+  private SpanContext remoteContext;
+  private Kind spanKind;
+  private List<Link> links;
+  private boolean recordEvents;
+  private Sampler sampler;
+  private final String spanName;
+  private boolean noParent;
+
+  SpanBuilderSdk(String spanName) {
+    this.spanName = spanName;
+  }
+
+  @Override
+  public Builder setParent(Span parent) {
+    this.parent = Utils.checkNotNull(parent, "parent");
+    this.noParent = true;
```

Comment:
Shouldn't this set noParent to false or just leave it untouched? If I get the API doc right noParent won't have to be evaluated anyway if a parent is set explicitly.

Reply:
This is a copy paste error, it should be set to `false`.